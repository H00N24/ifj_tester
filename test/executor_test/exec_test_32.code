// Pristup k premennej

class Main
{
	static int a = 5;
	static void run()
	{
		int a = 10;
		ifj16.print("a: " + a + " Main.a: " + Main.a + "\n");
	}
}