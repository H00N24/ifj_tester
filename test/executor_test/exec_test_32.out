a: 10 Main.a: 5
